cctbx/euclidean_model_matching.py

```diff
@@ -459,10 +465,17 @@ def get_transformed_model2(self,output_pdb=None,
         return model2.as_emma_model()
 
 def match_sort_function(match_a, match_b):
-  from past.builtins import cmp
-  i = -cmp(len(match_a.pairs), len(match_b.pairs))
-  if (i): return i
-  return cmp(match_a.rms, match_b.rms)
+  # Deprecated. Do not use
+  if len(match_a.pairs) < len(match_b.pairs):
+    return 1
+  elif len(match_a.pairs) > len(match_b.pairs):
+    return -1
+  if match_a.rms < match_b.rms:
+    return -1
+  elif match_a.rms > match_b.rms:
+    return 1
+  else:
+    return 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.